code/js/modules/BaseController.js

```diff
@@ -210,11 +216,23 @@
    * @return {{song: {String}, artist: {String}, isPlaying: {Boolean}, siteName: {String}}}
    */
   BaseController.prototype.getStateData = function() {
+    var vCanDislike = !!(this.selectors.dislike && this.doc().querySelector(this.selectors.dislike)),
+        vCanPlayPrev = !!(this.selectors.playPrev && this.doc().querySelector(this.selectors.playPrev)),
+        vCanPlayPause = !!((this.selectors.playPause && this.doc().querySelector(this.selectors.playPause)) || (this.selectors.play && this.doc().querySelector(this.selectors.play)) || (this.selectors.pause && this.doc().querySelector(this.selectors.pause))),
```

Comment:
We can remove the `canPlayPause` lines entirely. If a site does not support playPause then it will not be in the extension! This holds true now & for the future - playPause is the one absolutely necessary player feature :smiley: